File: INICIANTE/JAVASCRIPT/1165.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift(); };

var N = prompt("")

for(i = 0;i<N;i++){
    var saiu = 0
    var x = parseInt(prompt(""))
    for(y = 2;y < x;y++){
        if(x % y === 0){
            console.log(x + " nao eh primo")
            saiu = 1
            break;
        }
    }
    if(saiu === 0){
        console.log(x + " eh primo")
    }
}

File: INICIANTE/JAVA/1165.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int N = sc.nextInt();
        int x;
        
        for(int i = 0; i < N; i++){
            x = sc.nextInt();
            int saiu = 0;
            for(int y = 2; y < x; y++){
                if(x % y == 0){
                    System.out.println(x + " nao eh primo");
                    saiu = 1;
                    break;
                }
            }
            if(saiu == 0){
                System.out.println(x + " eh primo");
            }
        }
        sc.close();
    }
}
